ios/HeliumWidget/HeliumWidget.intentdefinition

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm unfamiliar with iOS intents, what is this used for?

Reply:
iOS intents are used to allow widget customizability by the consumer or user. For now that is just boilerplate and we can always add on to in the future. Here is some documentation on widget intents: https://developer.apple.com/documentation/widgetkit/making-a-configurable-widget